utils/build/docker/nodejs/parametric/servicer.js

```diff
@@ -65,10 +65,10 @@ class Servicer {
 
         const span = tracer.startSpan(request.name, {
             type: request.type,
-            resource: request.resource,
             childOf: parent,
             tags: {
-                service: request.service
+                service: request.service,
```

Comment:
I'm surprised this works as it should also be set to a different tag of `service.name`.

Reply:
oh! that might be why one of the tests is marked as expected to fail!